include/constants/vars.h

```diff
@@ -71,120 +71,120 @@
 #define VAR_DEOXYS_ROCK_STEP_COUNT           0x4034
 #define VAR_DEOXYS_ROCK_LEVEL                0x4035
 #define VAR_STORAGE_UNKNOWN                  0x4036
-#define VAR_0x4037                           0x4037
-#define VAR_0x4038                           0x4038
-#define VAR_0x4039                           0x4039
-#define VAR_0x403A                           0x403A
+#define VAR_UNUSUAL_WEATHER_LOCATION         0x4037
+#define VAR_UNUSUAL_WEATHER_STEP_COUNTER     0x4038
+#define VAR_SHOULD_END_UNUSUAL_WEATHER       0x4039
+#define VAR_NEVER_READ_0x403A                0x403A // Var is written to, but never checked
 #define VAR_REGICE_STEPS_1                   0x403B
 #define VAR_REGICE_STEPS_2                   0x403C
 #define VAR_REGICE_STEPS_3                   0x403D
 #define VAR_ALTERING_CAVE_WILD_SET           0x403E
-#define VAR_0x403F                           0x403F
+#define VAR_ALWAYS_ZERO_0x403F               0x403F // This var is read and written, but is always zero.
 #define VAR_DAYS                             0x4040
 #define VAR_FANCLUB_UNKNOWN_1                0x4041
 #define VAR_FANCLUB_UNKNOWN_2                0x4042
 #define VAR_DEPT_STORE_FLOOR                 0x4043
 #define VAR_0x4044                           0x4044
-#define VAR_POKELOT_PRIZE                    0x4045
+#define VAR_POKELOT_PRIZE_ITEM               0x4045
 #define VAR_NATIONAL_DEX                     0x4046
 #define VAR_SEEDOT_SIZE_RECORD               0x4047
 #define VAR_ASH_GATHER_COUNT                 0x4048
 #define VAR_BIRCH_STATE                      0x4049
 #define VAR_CRUISE_STEP_COUNT                0x404A
 #define VAR_POKELOT_RND1                     0x404B
 #define VAR_POKELOT_RND2                     0x404C
-#define VAR_0x404D                           0x404D
-#define VAR_0x404E                           0x404E
+#define VAR_POKELOT_PRIZE_PLACE              0x404D
+#define VAR_UNUSED_0x404E                    0x404E // Unused Var
 #define VAR_LOTAD_SIZE_RECORD                0x404F
 #define VAR_0x4050                           0x4050
```

Comment:
```suggestion
#define VAR_MOM_STATE                         0x4050
```

If you don't mind some help, I've seen this modified during the prologue when it relates to mom. When it's 3, she's standing outside ready to intercept the player as they leave town, for example.

Reply:
Yes; I've already documented all the intro sequence in #566 , but that PR was too big. I'm going to be opening a PR with all the intro event scripts completely documented when this is merged.